deployment/eks/flyte_aws_scheduler_helm_generated.yaml

```diff
@@ -836,14 +836,13 @@ metadata:
 spec:
   type: NodePort
   ports:
-  - name: http
-    port: 88
-    protocol: TCP
-    targetPort: 8088
```

Comment:
This was broken. Based on the config it actually listens on 8080: https://github.com/flyteorg/flyte/blob/master/charts/flyte-core/values.yaml#L953